Update Go documentation (#440)
developer-portal/content: tech/languages/go/go-programs.md

Hello, Fedora!
```

> Yet another option is to use `go install`. You still need to configure the modules as it was detailed before:

Reviewer: ```suggestion
Yet another option is to use `go install`. You still need to configure the modules as mentioned above:
```
